_stories/molecules/Breadcrumbs/index.js

```diff
@@ -66,39 +123,45 @@ class BreadcrumbsStory extends Component {
     static displayName = 'Breadcrumbs';
 
     state = {
-        hideSubmenus: false
+        hideSubmenus: false,
+        config: configA
+    };
+
+    changeConfig = () => {
+        const { config } = this.state;
+        const newConfig = config.path === 'home' ? configB : configA;
+        this.setState({ config: newConfig });
     };
 
+    printCode = code => '\n' + JSON.stringify(code, null, 4) + '\n\n';
+
     render() {
+        const { config } = this.state;
+        const configName = `Configuration ${config.path.includes('home') ? 'A' : 'B'}`;
         return (
-            <header className="gds-page-header">
-                <div className="gds-page-header__nav-bar">
-                    <Breadcrumbs
-                        config={config}
-                        pathname={select('pathname', routeOptions, routeOptions[0])}
-                        hideMenus={boolean('Hide Submenus', false)}
-                        hideRoot={boolean('Hide root breadcrumb', false)}
-                    />
-                </div>
-            </header>
+            <div>
+                <header className="gds-page-header">
```

Comment:
can you add `-color-bg-white` to the header. Currently content scrolled underneath displays through the breadcrumbs.

Reply:
Done